virtcontainers/hyperstart_agent.go

```diff
@@ -461,7 +478,18 @@ func (h *hyper) stopSandbox(sandbox *Sandbox) error {
 		return err
 	}
 
-	return h.proxy.stop(sandbox, h.state.ProxyPid)
+	if err := h.proxy.stop(h.state.ProxyPid); err != nil {
+		return err
+	}
+
+	h.state.ProxyPid = -1
+	h.state.URL = ""
+	if err := sandbox.storage.storeAgentState(sandbox.id, h.state); err != nil {
+		// ignore error
+		h.Logger().WithError(err).WithField("sandbox id", sandbox.id).Errorf("failed to clean up agent state")
```

Comment:
Can you make that field name just `sandbox` to avoid the space in it? Same comment for `kata_agent.go`.

Reply:
Good point!